tests/www/views/test_views_trigger_dag.py

```diff
@@ -369,6 +370,7 @@ def test_trigger_dag_params_array_value_none_render(admin_client, dag_maker, ses
 def test_dag_run_id_pattern(session, admin_client, pattern, run_id, result):
     with conf_vars({("scheduler", "allowed_run_id_pattern"): pattern}):
         test_dag_id = "example_bash_operator"
+        run_id = quote(run_id)
```

Comment:
This is only quoting in tests - there is no quoting in Airflow code . Did I miss somethig? Has this quoting been implmented since that bug fix?

Reply:
No. If you look at views.py change here, i replaced space with + which is now removed. That’s the bug fix. And it was introduced when the feature was added but should not have been there. The test was not properly done and it introduced the bug of changing space to + in code